tools/jira-release-version/main.go

```diff
@@ -13,19 +13,21 @@ import (
 )
 
 const (
-	envJira       = "JIRA_API_TOKEN"
-	envVersion    = "VERSION_NUMBER"
-	projectID     = "CLOUDP"
-	jiraURL       = "https://jira.mongodb.org"
-	versionPrefix = "terraform-provider-"
+	envJira         = "JIRA_API_TOKEN"
+	envVersion      = "VERSION_NUMBER"
+	projectKey      = "CLOUDP"
+	jiraURL         = "https://jira.mongodb.org"
+	versionPrefix   = "terraform-provider-"
+	versionNameNext = "next-terraform-provider-release"
 )
 
 func main() {
```

Comment:
[q] Curious if we were able to test this in any way, or if we will wait for a proper release.

Reply:
yes, i tested with a fake version

Reply:
adding an example of output in PR description